### ISSUE TITLE
dropdown choice does not "broadcast" to entire row/column

### ISSUE THREAD

**Issue by tjangoW** [QuantStack/ipysheet#114]
Hi,

I am running the following code (modified) and noticed that the `ipysheet.column` and `ipysheet.row` do not respect the `choice=tmp_dropdown` option.

```python
import ipysheet
sheet = ipysheet.sheet(rows=5, columns=5)
tmp_dropdown = ["1", "2", "3"]
cell1 = ipysheet.cell(1, 1, 0, type="dropdown", choice=tmp_dropdown)
cell2 = ipysheet.column(0, ["0"]*sheet.rows, type="dropdown", choice=tmp_dropdown)
cell3 = ipysheet.row(0, [0]*sheet.columns, type="dropdown", choice=tmp_dropdown)
print(sheet.cells)
sheet
```
The output of `print(sheet.cells)` is as follow:
```python
(Cell(choice=['1', '2', '3'], column_end=1, column_start=1, row_end=1, row_start=1, 
      type='dropdown', value=0),
 Cell(column_end=0, column_start=0, row_end=4, row_start=0, squeeze_row=False, 
      type='dropdown', value=['0', '0', '0', '0', '0']),
 Cell(column_start=0, row_end=0, row_start=0, squeeze_column=False, 
      type='dropdown', value=[1, 1, 1, 1, 1]))
```
Additional info about the output: the dropdown for 0th row and 0th column are of course not working, though the dropdown inverted triangle is shown as usual.

**Comment by martinRenou** [comment]
Thanks for filling this issue. I'll have a look at it.

**Comment by martinRenou** [comment]
This should be fixed by #116
